nginx-es/version.txt

```diff
@@ -1 +1 @@
-3.13.5-2
+3.13.5-3
```

Comment:
Just version update. No other change needed?

Reply:
I get it. Dockerfile already has the apk upgrade added to it.. just building a new version should include latest stable apk i suppose.